config/vufind/RecordDataFormatter.ini

```diff
@@ -0,0 +1,45 @@
+; This file can be used to set global default options for fields, to add new fields and to configure fields
+; in the RecordDataFormatter.
+
+; This section is used to set global default option.
+[Global]
+; Enable or disable all fields by default (default is true)
+;enabled = false
+; Use change default renderMethod (default is "Simple")
+;renderType = "CombineAlt"
+; Change the default separator for array values (default is <br />)
+;separator = "; "
+; One can also add any other option.
+
+; In this section, extra fields can be added that are not already included in the RecordDataFormatterFactory.
+; Add the name of the field to the respective array. The options of the field need to be set in
+; this file (see below for explanation).
+[Defaults]
+;collection-info[] =
+;collection-record[] =
+;core[] =
+;description[] =
+
+; Example:
+; core[] = "Extra"
+; core[] = "MySpecialField"
+; to add the two fields "Extra" and "MySpecialField" to core. See below for the configuration.
+
+; To change options on a specific field one has to add a section [Field_<field's name>].
+; E.g. to enable ISBNs and to add the separator option use:
```

Comment:
Maybe say "override" instead of "add" here?

Reply:
done